tests/Unit/Evolution/Systems/Cce/Test_WorldtubeData.cpp

```diff
@@ -311,7 +311,7 @@ class ReducedDummyBufferUpdater
 
   DataVector& get_time_buffer() noexcept override { return time_buffer_; }
 
-  bool radial_derivatives_need_renormalization() const noexcept override {
+  bool has_version_history() const noexcept override {
     return false;
```

Comment:
Well now the bool didn't change... so I don't know anymore.

None of this is a change request, just me seeing something unexpected and checking it's intentional.